easybuild/framework/easyblock.py

```diff
@@ -1445,7 +1445,20 @@ def extensions_step(self, fetch=False):
 
     def package_step(self):
         """Package software (e.g. into an RPM)."""
-        pass
+        rpmname = "HPCBIOS.20150211-%s-%s" % (self.name, self.version)
+        os.chdir(os.environ['TMPDIR'])
+        
+        
+        toolchaindep = "=".join([self.toolchain.name, self.toolchain.version])
+        if toolchaindep == "dummy=":
+            deplist = ""
+        else:
+            deplist = " ".join([" --dependency ", toolchaindep])
+        deplist += " ".join([" --dependency " + i 
+            for i in (map(lambda x:x["name"] + "=" + x["version"], self.cfg.dependencies()))
+        ])
+        cmd = "fpm --workdir $TMPDIR -t rpm --name %s -s dir %s -C %s" % (rpmname, deplist, self.installdir)
```

Comment:
@stdweird Isn't TMPDIR set by easybuild to be correct? I can't find a good way to grab it from python (the variable eb_tmpdir is set in main.py, but not into a parameter I think I can see from where package_fpm would run

looking at set_tmpdir in tools/config.py

```
    for var in ['TMPDIR', 'TEMP', 'TMP']:
        env.setvar(var, current_tmpdir)
```